pkg/alertmanager/alertmanager_http_test.go

```diff
@@ -0,0 +1,88 @@
+package alertmanager
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/alertmanager/cluster"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMultitenantAlertmanager_GetStatusHandler(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	peers := createAlertmanagerGossipCluster(ctx, t)
+
+	for _, tt := range []struct {
+		am        *MultitenantAlertmanager
+		content   string
+		nocontent string
+	}{
+		{
+			am:        &MultitenantAlertmanager{peer: nil},
+			content:   "Alertmanager gossip clustering is disabled",
+			nocontent: "Node",
+		},
+		{
+			am:        &MultitenantAlertmanager{peer: peers[1]},
+			content:   "Members",
+			nocontent: "No peers",
+		},
+	} {
+		req := httptest.NewRequest("GET", "http://alertmanager.cortex/status", nil)
+		w := httptest.NewRecorder()
+		tt.am.GetStatusHandler().ServeHTTP(w, req)
+
+		resp := w.Result()
+		require.Equal(t, 200, w.Code)
+		body, _ := ioutil.ReadAll(resp.Body)
+		content := string(body)
+		require.Contains(t, content, tt.content)
+		require.NotContains(t, content, tt.nocontent)
+	}
+}
+
+func createAlertmanagerGossipCluster(ctx context.Context, t *testing.T) []*cluster.Peer {
```

Comment:
Here you create 2 peers, but the 2nd one is never used. You can simplify this helper just creating and returning 1 peer.